packages/amplify-category-analytics/provider-utils/awscloudformation/service-walkthroughs/kinesis-walkthrough.js

```diff
@@ -0,0 +1,208 @@
+const inquirer = require('inquirer');
+const path = require('path');
+// FIXME: may be removed from here, since addResource can pass category to addWalkthrough
```

Comment:
Leftover to fix?

Reply:
more a message for you guys for the future, I mean the maintainers, I saw across overall amplify categories `provider-utils/awscloudformation/index.js` would hardcode the category name, while it is generally available in root `amplfy-category-.....`/index.js so passing it down feels just slightly cleaner, it's minor anyway, do you want me to remove it?

Reply:
If you notice definitions in https://github.com/ambientlight/amplify-cli/blob/lambda-custom-event-source/packages/amplify-category-analytics/provider-utils/awscloudformation/index.js take category as parameter which is unused which made me feel it was an intention